SPECS/ipv6calc/ipv6calc.spec

```diff
@@ -49,12 +40,15 @@ Provides:       ipv6calc-libs = %{version}-%{release}
 %else
 Conflicts:      ipv6calc-libs
 %endif
-
 %if %{require_db4}
 BuildRequires:  db4-devel
 %else
 BuildRequires:  libdb-devel
```

Comment:
Just curious, since we've been trying to decrease our `libdb` usage- is there another database we can use with this package?

Reply:
Will leave for now since it looks like Fedora is poking upstream to support perl-lmdb (https://fedoraproject.org/wiki/User:Pkubat/Draft_-_Removing_BerkeleyDB_from_Fedora). Marking internally that we should check back in on this.